src/SystemLoader.cc

```diff
@@ -110,10 +110,10 @@ class ignition::gazebo::SystemLoaderPrivate
   }
 
   // Default plugin search path environment variable
-  public: std::string pluginPathEnv{"IGN_GAZEBO_SYSTEM_PLUGIN_PATH"};
+  public: std::string pluginPathEnv{"GZ_SIM_SYSTEM_PLUGIN_PATH"};
```

Comment:
We need to deprecate this like in https://github.com/gazebosim/gz-gui/pull/384

Reply:
I took your advice and reverted the env vars for `gz-sim` specifically.
https://github.com/gazebosim/gz-sim/pull/1496/commits/39c3200f39e73f66d4e2d23d1753e1531012e1b2